scripts/download

```diff
@@ -20,7 +20,11 @@ rm -rf ${CONTAINERD_DIR}
 mkdir -p ${CHARTS_DIR}
 mkdir -p ${DATA_DIR}
 
-curl --compressed -sfL https://github.com/k3s-io/k3s-root/releases/download/${VERSION_ROOT}/k3s-root-${ARCH}.tar | tar xf -
+if [ ${ARCH} = riscv64 ]; then
+  curl --compressed -sfL https://github.com/CARV-ICS-FORTH/k3s-root/releases/download/20230614/k3s-root-riscv64.tar | tar xf -
+else
+  curl --compressed -sfL https://github.com/k3s-io/k3s-root/releases/download/${VERSION_ROOT}/k3s-root-${ARCH}.tar | tar xf -
+fi
```

Comment:
You can now drop this and bump VERSION_ROOT to `v0.13.0`:
https://github.com/k3s-io/k3s-root/releases/tag/v0.13.0